guides/policy/azure-authenticator/parseAzureAuthenticator.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+    $(document).ready(function () {
+
+        $('#encodedToken').on('input', tokenInputChanged);
+
+        $('#tblInput input').on('input', hostIdInputChanged);
+    });
+
+    let hostId;
+    let encodedToken;
+
+    function hostIdInputChanged() {
+        hostId = $("#txtHostId").val();
+        updatePolicy();
+    }
+
+    function tokenInputChanged() {
+        encodedToken = $("#encodedToken").val();
+        updatePolicy();
+    }
+
+    function updatePolicy() {
+        let output = "";
+        if (!hostId) {
+            output = "host ID is missing";
+        } else if (!encodedToken) {
+            output = "Azure AD token is missing";
+        } else {
+            output = getPolicyFromToken(encodedToken, hostId);
+        }
+        $("#output").html(output);
+    }
+
+    function getPolicyFromToken(encodedToken, appId) {
+        try {
+            let decodedToken = jwt_decode(encodedToken);
+            let token = parseAzureToken(decodedToken);
+
+            return `
+            - !host
+              id: ${hostId}
+              annotations:
+                authn-azure/subscription-id: ${token.subscriptionId}
+                authn-azure/resource-group: ${token.resourceGroup}
+              # The following is optional
+                authn-azure/${token.isUserIdentity ? "user" : "system"}-assigned-identity: ${token.identity}
+            `;
+        } catch (e) {
+            console.error(e);
+            return "Failed to parse Azure AD Token"
```

Comment:
Consider showing the error... Think about the flow of a customer that got the "failed to parse" message...
He goes to the repo and open an issue with this text... now what are we need to do to support him?